app/controllers/restaurants_controller.rb

```diff
@@ -3,6 +3,11 @@ class RestaurantsController < ApplicationController
 
   def index
     @restaurants = Restaurant.all
```

Comment:
Hi Justin. We were planning on implementing functionality of the search bar in react, since our restaurants#index page is completely react. In that case, capybara tests are not possible. Would you prefer we not include the searchbar at all then for this PR?